webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/services/DisplayNode.scala

```diff
@@ -633,6 +636,14 @@ object DisplayNode extends Loggable {
       DateFormaterService.getDisplayDate(nodeFact.factProcessedDate)
     }
           </div>
+          {
+      agentRun
+        .map(agentRunWithNodeConfig =>
+          <div><label>Agent last run:</label> {DateFormaterService.getDisplayDate(agentRunWithNodeConfig.agentRunId.date)}</div>
+        )
+        .getOrElse(NodeSeq.Empty)
```

Comment:
I think we always want to have `<div><label>Agent last run:` but with a "none available" or something like that when we don't have any. It's better for the user to know that the information is not here (but the UI is always the same) than to wonder why the info he is looking for is not at the usual place.